meta-iotqa/lib/oeqa/runtime/multimedia/opencl/opencl.py

```diff
@@ -0,0 +1,10 @@
+from oeqa.oetest import oeRuntimeTest
+
+class OpenCLTest(oeRuntimeTest):
+    def test_opencl_has_library_file(self):
+        (status, output) = self.target.run('ls /usr/lib/libOpenCL*')
```

Comment:
@epyeoh I don't see the feedback from @ipuustin is addressed. Do you plan to update the PR accordingly?